src/Nethermind/Nethermind.Consensus.Clique/CliquePlugin.cs

```diff
@@ -192,6 +192,9 @@ public Task InitRpcModules()
         }
 
         public string SealEngineType => Nethermind.Core.SealEngineType.Clique;
+        
+        //TODO: redo clique producer
```

Comment:
ToDo?

Reply:
Yes, wanted to indicate this will not work with Clique properly yet